### PR TITLE
chore(threshold)

### PR DESCRIPTION
**Pull request by jjBlockchain**
update threshold to 10 and use v5 link from env variable

### CODE REVIEW

**Comment by Sentini14** [comment]
Yes it’s me I need to move my crypto

***@***.***

**Review by jhagerman-bc — APPROVED** [review]

**Review by Sentini14 — COMMENTED** [review]
I want my wallet address important

**Comment by Sentini14** [comment]
Approved.

**Comment by sstephanou-bc** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [5c8e37c3\-6768\-4890\-8274\-8142d2501520](https://eu.ast.checkmarx.net/projects/f3c879d8-da6e-4aaa-aeee-f377cc306d79/scans?id=5c8e37c3-6768-4890-8274-8142d2501520&branch=chore%2Fthreshold)
### No New Or Fixed Issues Found

**Comment by Sentini14** [comment]
Fixed!

**Review by mperdomo-bc — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/scenes/app.tsx [hunk 1]
```diff
@@ -202,8 +202,7 @@ const App = ({
       // eslint-disable-next-line
       console.log('Redirecting to v5')
       // Using **WALLET_V5_LINK** as a fallback for webpack builder.
```
Comment (Sentini14) [inline packages/blockchain-wallet-v4-frontend/src/scenes/app.tsx:204]: Hips ca I login

**Pull request merged**